cinder/volume/drivers/vmware/volumeops.py

```diff
@@ -389,6 +391,7 @@ def reload_backing(self, backing):
 
     # TODO(kartikaditya) Keep the methods not specific to volume in
     # a different file
+    @cached(cache=TTLCache(maxsize=128, ttl=300))
```

Comment:
This could easily change in 5min, because DRS runs every 5min. I'm not sure this is safe to cache.